tests/test/ir_cache/test_ir_registry.cpp

```diff
@@ -45,15 +45,10 @@ namespace tests {
         REQUIRE(moduleRefA1.tables.defs[0].type.size.max == MAX_TABLE_SIZE);
```

Comment:
Reformatted this as part of my clang-format experimentation. I'll do a single PR to run it across the whole codebase soon.